src/lib/components/navigation.svelte

```diff
@@ -1,21 +1,21 @@
 <script lang="ts">
-  import type { Lesson } from "$lib/types";
+  import type { Page } from "$lib/types";
```

Comment:
Made this clearer - we now have a Lesson type that is exactly a Lesson as defined in course-structure, and 'Page' which has a previousSlug / nextSlug etc.